app/controllers/use_case_controller.rb

```diff
@@ -3,6 +3,7 @@
 # * permitted_params
 class UseCaseController < ApplicationController
   before_filter :authenticate_user!
+  before_filter ->{authorize self.model_class.new}, only: [:new, :create]
```

Comment:
I thought you could collapse this into the following line, but the object doesn't exist yet, my mistake.

Authorizing on an empty object still seems suspicious. Would this work: `authorize self.model_class`?